how-to/workspace-platform-starter/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Added support for Snap, enable by setting `customSettings.snapProvider.enabled` to true. Configure the `customSettings.snapProvider.serverAssetInfo` to point to the `SNAP_ASSET_URL`. Enable the Snap debugging window by setting `customSettings.snapProvider.showDebugWindow` to true.
 - Added new module type `content-creation`, these modules can be used to define content creation rules and handle the associated events. Modules are added in `customSettings.contentCreationProvider` section in manifest.
 - Added example content creation module which interrogates the `features` property from `window.open` to determine where to place a view in relation to where it was launched from. An example app `Content Creation Example` demonstrates this in use.
+- Change moved pin/unpin/move-view/move-window actions in to module
```

Comment:
I would add a snippet that shows the thing they have to add to the manifest (It's in the commit but people may look at this and not the commit) and just say to keep the existing menu options you will need to do this.